RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -107,6 +117,7 @@ import RevenueCat
         self.actionWrapper = actionWrapper
         self.purchasesProvider = purchasesProvider
         self.customerCenterStoreKitUtilities = customerCenterStoreKitUtilities
+        self.customerInfo = nil
```

Comment:
Hmm this is always set to nil right? Seems they are unused?

Reply:
I missed to push one last commit 🙃 
There it is, it's assigned at the beginning of `loadPurchaseInformation`